master/maintenance/kubernetes-upgrade.md

```diff
@@ -157,8 +149,4 @@ and your datastore type.
    upgrade by running `calico-upgrade complete`. After this, you can once again schedule
```

Comment:
I don't think this block is relevant any more since we've removed the v2.6 -> v3 upgrade content.

Reply:
Also wonder if we should explicitly call out at the top of this page that it covers upgrade between Calico v3.x versions only.